src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1917,6 +1919,12 @@ def server_create(
     if assign_identity:
         kwargs['identity'] = ResourceIdentity(type=IdentityType.system_assigned.value)
 
+    if enable_public_network is not None:
+        if enable_public_network:
+            kwargs['public_network_access'] = ServerPublicNetworkAccess.enabled
```

Comment:
suggestion: `kwargs['public_network_access'] = ServerPublicNetworkAccess.enabled if enable_public_network else ServerPublicNetworkAccess.disabled`

Reply:
similar below

Reply:
yes, `A if X else B` is weird syntax but that's python :p